package.json

```diff
@@ -31,7 +31,7 @@
     "raygun4js": "2.18.0",
     "react": "16.10.2",
     "react-dom": "16.10.2",
-    "react-intl": "2.9.0",
+    "react-intl": "3.3.2",
```

Comment:
When updating `react-intl` we should also update `@types/react-intl`.

Reply:
We can't do that until we moved all components to functional components. Reason for that is that react-intl moved the types to the package itself and the 3.x types are not working anymore.

Reply:
Also the temporary hack for using the hook won't work.